packages/block-editor/src/hooks/style.js

```diff
@@ -259,17 +257,23 @@ const withElementsStyles = createHigherOrderComponent(
 
 		return (
 			<>
-				<style
-					dangerouslySetInnerHTML={ {
-						__html: styles,
-					} }
-				/>
+				{ elements && (
+					<style
```

Comment:
Aside: (something for other PRs) Also noting that inline styles like that (which we do in layout as well), can cause CSS issues due to the new elements breaking `:first-child` selectors and things like that in editor styles.

@jorgefilipecosta @nosolosw @ellatrix We may need a dedicated way to inject these styles in the editor (and maybe even in the frontend) that doesn't suffer from these issues.